.github/workflows/build-test.yml

```diff
@@ -281,6 +285,13 @@ jobs:
         name: wheels
         path: ./target/wheels/
 
+    - name: Check for incompatible wheels and bail if found
+      run: |
+        set -e
+        # Check for a wheel with -linux_x86_64 platform tag, which will
+        # fail uploading to pypi.
+        find target -name \*-linux_x86_64.whl -exec /bin/false '{}' '+'
```

Comment:
I don't think this is needed, personally.  Or if it is, we ought to use some complete check across all platforms using some Python tooling.

Reply:
I don't think it'll hurt anything and it'll keep the same thing from happening.  If it doesn't do any harm i'm inclined to leave more checks in and remove them if they become a problem for some reason.